modules/aws-eks-self-managed-node-groups/locals.tf

```diff
@@ -62,13 +68,24 @@ locals {
     local.self_managed_node_group["additional_iam_policies"
   ]))
 
+  k8s_labels = merge(
+    local.self_managed_node_group["k8s_labels"],
+    {
+      "eks/capacityType"    = local.self_managed_node_group["capacity_type"],
+      "eks/node_group_name" = local.self_managed_node_group["node_group_name"]
+    }
+  )
+
```

Comment:
Are(`k8s_labels`) these being applied anywhere in the code?

Reply:
@vara-bonthu I'm replicating what MGNs are doing. The `eks/capacityType` tag will be used to schedule pods in these nodes, and the `eks/node_group_name` is used to help CA identify which node groups to scale (we need this when the node groups wants to scale down to zero).